### ISSUE TITLE
Steam Access Denied

### ISSUE THREAD

**Issue by rayman3003** [therealaleph/MasterHttpRelayVPN-RUST#1097]
When trying to open the steam website, it gives me the following error:

"Access Denied
You don't have permission to access "http://store.steampowered.com/" on this server.
Reference #18.4fa42617.1778589519.63ec7f32

https://errors.edgesuite.net/18.4fa42617.1778589519.63ec7f32"

"Uh oh! Something went wrong with the previous request.
If you navigated here, it's because the previous request experienced an error. If you are the site owner, you can use the Translate Error String tool to investigate this further. This is located on the control center under:

Hamburger menu > SUPPORT > Edge Diagnostics."

How to solve it?

**Comment by therealaleph** [comment]
Reviewed via Anthropic Claude.

@rayman3003 — `Access Denied / errors.edgesuite.net / Reference #18.` is **Akamai's anti-bot block**. Steam routes through Akamai's edge, and Akamai flags Apps Script's Google datacenter IPs as bot-prone.

Fix: route Steam through an exit-node (Deno Deploy / Replit / VPS) instead of Apps Script. The exit-node uses a residential-looking IP that Akamai doesn't flag.

Add to `config.json`:
```
"exit_node": {
  "enabled": true,
  "relay_url": "https://your-exit-node",
  "psk": "<secret>",
  "mode": "selective",
  "hosts": ["store.steampowered.com", "steamcommunity.com", "checkout.steampowered.com", "login.steampowered.com"]
}
```

Setup exit-node: [`assets/exit_node/README.md`](https://github.com/therealaleph/MasterHttpRelayVPN-RUST/blob/main/assets/exit_node/README.md). For Iranian users who can't sign up for Deno Deploy, alternatives at #908 (Replit, HuggingFace Spaces, friend-with-VPS).

PR #963 (utls Chrome ClientHello fingerprinting, in community testing) will also fix this once merged — utls makes mhrv-rs's outbound TLS look like Chrome at the fingerprint level, which sidesteps Akamai's JA3 check entirely.

Closing — exit-node is the path today, PR #963 is the structural fix.

**Comment by rayman3003** [comment]
@therealaleph 

I live in ran, & none of the solutions that mention on https://github.com/therealaleph/MasterHttpRelayVPN-RUST/issues/908 r working for me:

1- I don't have any working VPN; & Deno won't allows for sign-up. When trying to signup, it says:

Sorry, there was an issue loading this page
401 NOT_AUTHENTICATED 🔐
The request was not authenticated.
Trace ID:
b9c8d184b24852d2772d2932e3e8329e

2- When trying to opening the replit webpage for signing-up, it says:

Sorry, you have been blocked
You are unable to access replit.com

3- When trying to signing-up to HuggingFace Spaces (Docker), it says:

403 ERROR
The request could not be satisfied.
Request blocked. We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.
Generated by cloudfront (CloudFront)
Request ID: GmgwEDvFmMFjlarOkCm8jIpJji7ifM_ZMy0_G-ZLRXJwtkdryue0lQ==

5- I don't have any friends that has VPS.
